control_panel_api/tests/test_models.py

```diff
@@ -0,0 +1,79 @@
+from django.test import TestCase
+
+from control_panel_api.models import (
+    Role,
+    Team,
+    TeamMembership,
+    User,
+)
+
+
+class FixtureTestCase(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        # Users
+        cls.user_alice, _ = User.objects.get_or_create(username="Alice")
+        cls.user_bob, _ = User.objects.get_or_create(username="Bob")
+        cls.user_other, _ = User.objects.get_or_create(username="Other")
+        # Teams
+        cls.team_justice, _ = Team.objects.get_or_create(name="Justice Team")
+        cls.team_other, _ = Team.objects.get_or_create(name="Other Team")
+        # Roles
+        cls.role_maintainer, _ = Role.objects.get_or_create(
+            code="maintainer",
+            name="Maintainer",
+        )
+        cls.role_member, _ = Role.objects.get_or_create(
+            code="member",
+            name="Member",
+        )
+
+        # Add Alice to Justice team as maintainer
+        TeamMembership.objects.create(
+            team=cls.team_justice,
+            user=cls.user_alice,
+            role=cls.role_maintainer,
+        )
+        # Add Bob to Justice team as member
+        TeamMembership.objects.create(
+            team=cls.team_justice,
+            user=cls.user_bob,
+            role=cls.role_member,
+        )
+        # Add other user to other team as member
+        TeamMembership.objects.create(
+            team=cls.team_other,
+            user=cls.user_other,
+            role=cls.role_member,
+        )
```

Comment:
These `TeamMembership` objects should not be in a global setup - these are setting up the context for individual tests and should be created in the tests methods that test them. Eg:
```python
def test_alice_is_in_team_justice(self):
    TeamMembership.objects.create(user=alice, team=team_justice, role=maintainer)
    self.assertIn(team_justice, alice.teams())
    self.assertNotIn(team_other, alice.teams())
```

The first set of fixtures (alice, bob, other, and the teams) are like the "Given some users and teams exist" context.
The second set are more like "When Alice is maintainer of Team Justice".